_posts/2018-01-10-cockpit-159.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Cockpit 159
+author: pitti
+date: 2018-01-10 10:30
+tags: cockpit linux
+slug: cockpit-159
+category: release
+summary: Cockpit with VDO support and VM improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 159.
+
+### Support VDO devices on Storage page
```

Comment:
Configure data deduplication with VDO devices on Storage page

Reply:
Good point. I left out the "Storage page" bit in exchange to avoid the heading becoming too long. That's that's already mentioned in the detailed description, after all.